src/SelfUpdateCommand.php

```diff
@@ -198,28 +233,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
             unset($phar);
             @rename($tempFilename, $localFilename);
             $output->writeln('<info>Successfully updated ' . $programName . '</info>');
-            $this->_exit();
+
+            exit;
```

Comment:
Methods that call `exit` are untestable. I'd prefer to keep the `_exit()` wrapper even if we aren't trying to run unit tests on this method.

Reply:
Restored